opus/application/static_media/js/browse.js

```diff
@@ -1235,7 +1241,8 @@ var o_browse = {
             $(selector).on("request.infiniteScroll", function(event, path) {
                 // hide default page status loader if op-page-loading-status loader is spinning
                 // && o_browse.tableSorting
-                $(".infinite-scroll-request").hide();
+                // Comment this out so that we will have spinner displayed
+                // $(".infinite-scroll-request").hide();
```

Comment:
is this going away forever....?

Reply:
I keep those codes commented because those codes were there for B061. But in startobs branch, that line of code breaks the infiniteScroll spinner. For now, I comment it out, and will revisit B061 to confirm if we can remove those codes once infiniteScroll issue is fixed.